### PR TITLE
Fix/table classes

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -1,212 +1,193 @@
 <div class="form-group" ng-class="{'has-error': !displayDetails.name.$pristine && displayDetails.name.$invalid}" show-errors>
-  <label class="control-label">{{'displays-app.fields.name.name' | translate}} *</label>
```
Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @RV-Adrian lots of spacing changes in this file. Not sure if there are any actual code changes in here as it is almost impossible to review, can you confirm? Also, are the formatting updates necessary? Thanks.

File: web/partials/editor/presentation-list.html [hunk 2]
```diff
@@ -20,41 +20,41 @@ <h1 id="title" translate>editor-app.list.title</h1>
...
       rv-spinner-start-active="1"
       >
-      <table id="presentationListTable" class="table-2 table-hover">
-        <thead>
-          <tr class="clickable">
-            <th id="tableHeaderName" ng-click="factory.sortBy('name')">
+      <table id="presentationListTable" class="table table--hover">
+        <thead class="table-header">
+          <tr class="table-header__row clickable">
+            <th id="tableHeaderName" class="table-header__cell clickable" ng-click="factory.sortBy('name')">
```
Comment (alex-deaconu) [inline web/partials/editor/presentation-list.html]: @RV-Adrian curious, is the `clickable` class necessary here as the `<tr>` already has it? Thanks.

Comment (RV-Adrian) [inline web/partials/editor/presentation-list.html]: @alex-deaconu Good point.  I ended up adding 'clickable' to rows only on every table, and that seems to have done the trick and cleaned up class usage a bit.

File: web/partials/displays/display-fields.html (hunk 1 above)
Comment (RV-Adrian) [inline web/partials/displays/display-fields.html]: @alex-deaconu It appears to be only formatting. Not sure why the spacing got thrown in. Is there a standard Apps usually uses to get on the same page with indentation etc? (ie. a certain Sublime package)

Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @RV-Adrian no we don't use a standard, but generally formatting changes are very hard to review and mostly unnecessary in a commit. Are you using a plugin that made these changes automatically? Thanks.

Comment (RV-Adrian) [inline web/partials/displays/display-fields.html]: @alex-deaconu I use a prettifier shortcut in Sublime. I may have accidentally hit it in this HTML file. Formatting adjusted now, should be easier to scan.

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good to me 👍 . Thanks!

**Pull request merged**